utils/hwloc/Makefile.am

```diff
@@ -66,19 +67,37 @@ hwloc_ps_SOURCES = hwloc-ps.c
 # keep HWLOC_PS_LIBS first in case there's also -lnsl which cannot be before -lsocket
 hwloc_ps_LDADD = $(HWLOC_PS_LIBS) $(LDADD) libutils_common.la
 
+noinst_LTLIBRARIES+=libhwloc-tbind.la
+libhwloc_tbind_la_SOURCES=hwloc-tbind-obj.c hwloc-tbind.h
+LIBDADD=$(HWLOC_top_builddir)/hwloc/libhwloc.la
+
+hwloc_tbind_SOURCES=hwloc-tbind.c hwloc-tbind.h
+LDADD+=$(HWLOC_top_builddir)/hwloc/libhwloc.la
+LDADD+=libhwloc-tbind.la
+
+AM_CFLAGS+=-DTBIND_XML_DIR=\"$(HWLOC_top_builddir)/tests/hwloc/xml\"
```

Comment:
`-D` options should go in to `AM_CPPFLAGS`.

Reply:
Done.  see #https://github.com/open-mpi/hwloc/pull/359/commits/a40b3ebdf08136431805ac871ca6f895a3452d64